Octokit/Clients/IRepoCollaboratorsClient.cs

```diff
@@ -59,6 +59,16 @@ public interface IRepoCollaboratorsClient
         Task Add(string owner, string repo, string user);
 
         /// <summary>
+        /// Invites a new collaborator to the repo
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="http://developer.github.com/v3/repos/collaborators/#add-collaborator">API documentation</a> for more information.
+        /// </remarks>
+        /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>
+        /// <returns><see cref="Task"/></returns>
+        Task<RepositoryInvitation> Invite(string owner, string repo, string user);
```

Comment:
According to the API docs, this function should actually take a request object stating the `permissions` for the user to  have.  This is missing from the `Add()` method which you would have copied for the `Invite()` method (an issue is raised for that but hasn't been fixed yet).  But we may as well at least include the `permissions` request on the `Invite()` method